src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,82 @@
-import React from 'react';
-import { NewCommentForm } from '../NewCommentForm';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+import { NewCommentForm } from '../NewCommentForm';
+
+import { getPostById } from '../../api/posts';
+import { getPostComments, deleteComment } from '../../api/comments';
+
+export const PostDetails = ({ postId }) => {
+  const [post, setPost] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [hideComments, setHideComments] = useState(false);
```

Comment:
if it's a boolean, then name it appropriately
I recommend you to read this short guide on naming convention
https://github.com/kettanaito/naming-cheatsheet

```suggestion
  const [isCommentsHidden, setIsCommentsHidden] = useState(false);
```